learning/tour-of-beam/frontend/lib/components/expansion_tile_wrapper.dart

```diff
@@ -0,0 +1,22 @@
+import 'package:flutter/material.dart';
+
+class ExpansionTileWrapper extends StatelessWidget {
+  final ExpansionTile expansionTile;
+  const ExpansionTileWrapper({required this.expansionTile});
+
+  @override
+  Widget build(BuildContext context) {
+    return Theme(
+      data: Theme.of(context).copyWith(
+        hoverColor: Colors.transparent,
+        splashColor: Colors.transparent,
+        highlightColor: Colors.transparent,
+        dividerColor: Colors.transparent,
+        unselectedWidgetColor: Colors.grey,
+        colorScheme: ColorScheme.fromSwatch(primarySwatch: Colors.grey),
+        visualDensity: const VisualDensity(vertical: -4),
```

Comment:
Can you find a way to also set `ListTileTheme.dense = true`? This will make it the shortest possible.

Reply:
Fixed.